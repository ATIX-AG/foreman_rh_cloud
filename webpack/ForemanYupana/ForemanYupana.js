import React from 'react';
import { ButtonGroup, Button } from 'patternfly-react';

const ForemanYupana = () => (
    <div style={{ paddingTop: '10px', paddingBottom: '10px' }}>
      <ButtonGroup bsSize="large">
        <Button bsStyle="link">index-tasks-page</Button>
        <Button bsStyle="link">show-task-page</Button>
      </ButtonGroup>
    </div>
);

export default ForemanYupana;
