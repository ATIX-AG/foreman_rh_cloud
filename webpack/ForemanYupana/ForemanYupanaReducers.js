import { combineReducers } from 'redux';
import { reducers as tasksDashboardReducers } from './Components/TasksDashboard';

const reducers = {
  ForemanYupana: combineReducers({
    ...tasksDashboardReducers,
  }),
};

export default reducers;
