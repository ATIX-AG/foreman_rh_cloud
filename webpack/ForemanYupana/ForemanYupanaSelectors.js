export const selectForemanYupana = state => state.ForemanYupana || {};
