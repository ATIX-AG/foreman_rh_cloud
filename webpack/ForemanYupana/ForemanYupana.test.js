import { testComponentSnapshotsWithFixtures } from 'react-redux-test-utils';

import ForemanYupana from './ForemanYupana';

const fixtures = {
  'render without Props': {},
};

describe('ForemanYupana', () =>
  testComponentSnapshotsWithFixtures(ForemanYupana, fixtures));
