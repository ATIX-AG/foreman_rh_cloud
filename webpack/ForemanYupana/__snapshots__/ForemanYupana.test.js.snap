// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForemanYupana render without Props 1`] = `
<div
  style={
    Object {
      "paddingBottom": "10px",
      "paddingTop": "10px",
    }
  }
>
  <ButtonGroup
    block={false}
    bsClass="btn-group"
    bsSize="large"
    justified={false}
    vertical={false}
  >
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="link"
      disabled={false}
    >
      index-tasks-page
    </Button>
    <Button
      active={false}
      block={false}
      bsClass="btn"
      bsStyle="link"
      disabled={false}
    >
      show-task-page
    </Button>
  </ButtonGroup>
</div>
`;
